satpy/etc/composites/mwr.yaml

```diff
@@ -0,0 +1,46 @@
+sensor_name: mwr
```

Comment:
This is a microwave instrument right? Do we need to define a "base" sensor type for "microwave" making this `sensor_name: microwave/mwr`? Or does this work as-is and we won't worry about a shared set of "microwave" composites? If this isn't microwave, then this should probably be `sensor_name: visir/mwr`, right?